.github/workflows/tests.yaml

```diff
@@ -22,6 +22,7 @@ jobs:
           - {name: 'PyPy', python: pypy3, os: ubuntu-latest, tox: pypy3}
           - {name: Style, python: '3.8', os: ubuntu-latest, tox: style}
           - {name: Docs, python: '3.8', os: ubuntu-latest, tox: docs}
+          - {name: Typing, python: '3.8', os: ubuntu-latest, tox: mypy}
```

Comment:
This shouldn't be here.

Reply:
Okay, done. Anything else? Because, really, I just copied the entire folder and did search+replace for "Werkzeug".